plugins/PrepareStage/PrepareMenu.qml

```diff
@@ -120,24 +112,12 @@ Item
                 id: background
                 height: UM.Theme.getSize("stage_menu").height
                 width: UM.Theme.getSize("stage_menu").height
+                border.color: UM.Theme.getColor("lining")
+                border.width: UM.Theme.getSize("default_lining").width
 
                 radius: UM.Theme.getSize("default_radius").width
                 color: openFileButton.hovered ? UM.Theme.getColor("action_button_hovered") : UM.Theme.getColor("action_button")
             }
-
-            DropShadow
```

Comment:
Removing the DropShadow means that the QtGraphicalEffects import can be removed too.

Reply:
Which is a good thing, since QtGraphical effects isn't supported by Qt 6.x (and might never be?)

Reply:
Good catch!